chia/cmds/coins.py

```diff
@@ -216,22 +207,22 @@ def combine_cmd(
     type=str,
     required=True,
 )
-@click.option("-t", "--target-coin-id", type=str, required=True, help="The coin id of the coin we are splitting.")
+@click.option("-t", "--target-id", type=str, required=True, help="The coin id of the coin we are splitting.")
```

Comment:
Does `-id` distinguish anything here?  Almost makes it sound like we're saying it's not a coin but an id instead.